CONTRIBUTING.md

```diff
@@ -1,9 +1,73 @@
 # How to Contribute
 
-We'd love to accept your patches and contributions to this project soon. But we
-are not yet ready to accept community contributions at this time.
+Everyone is welcome to contribute to MLIR. There are several ways of getting involved and contributing including reporting bugs, improving documentation, writing models or tutorials. 
+
+Please read our our [Code of Conduct](https://github.com/tensorflow/swift/blob/master/CODE_OF_CONDUCT.md) before participating.
```

Comment:
Should we check a copy here instead of referencing to another repo?

Reply:
I've updated to point to TF. I don't want to create another copy, else we'll run the risk of getting out of sync.